app/models/job.rb

```diff
@@ -1,4 +1,18 @@
 class Job < ActiveRecord::Base
   validates :title, :level_of_interest, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments
+
+  def self.level_of_interest
+     group(:level_of_interest).count
+  end
+
+  def self.sorted_by_interest_level
```

Comment:
i suggest breaking up the sorting by interest level and the grabbing the top three. this way, if you want to reuse the list of sorted jobs, you can!